app/controllers/submissions_controller.rb

```diff
@@ -718,6 +718,16 @@ def get_all_file_data(revision, grouping, path)
         data[:key] = file_name
         data[:modified] = data[:last_revised_date]
         data[:revision_by] = '' if anonymize
+        if data[:type] == 'unknown'
```

Comment:
It's a bit awkward to check this here, modify `get_file_info` above instead.

Reply:
Misha suggested a simple solution for this instead of modifying the get_file_info method and making it more complex. I have updated the PR based on that.